index.html

```diff
@@ -2601,17 +2626,498 @@ <h4>Derived Proof</h4>
+and comes from the blind BBS proof generation function which takes as inputs
+the |adjSelectiveIndexes|.
+          </p>
+          <pre class="example nohighlight"
+          title="Disclosure Data for Holder Binding"
+          data-include="TestVectors/HolderBinding/derivedDisclosureData.json"
+          data-include-format="text"></pre>
+          <p>
+Finally, using the disclosure data above with the algorithm of Section
+<a href="#serializederivedproofvalue"></a>, we obtain the signed derived
+(reveal) document shown below.
+          </p>
+          <pre class="example nohighlight"
+          title="Signed Derived Document for Holder Binding"
+          data-include="TestVectors/HolderBinding/derivedRevealDocument.json"
+          data-include-format="text"></pre>
+          </section>
+        </section>
+        <section>
+          <h4>Pseudonym with Issuer-known PID Feature</h4>
+          <section>
+            <h5>Issuer PID Base Proof</h5>
+            <p>
+To issue a credential under the <em>pseudonym with issuer-known PID</em> feature
+the issuer generates a cryptographically random |pid| value for a holder. This
+value is shared with the holder but is otherwise kept secret. This value is
+shown below.
+          </p>
+          <pre class="example nohighlight" title="Issuer Generated PID"
+          data-include="TestVectors/FeatureInputs/issuerPid.json"
+          data-include-format="text"></pre>
+          <p>
+This example will make use of the same key material shown in
+<a href="#example-private-and-public-keys-for-signature-0"></a>. It will
+utilize the same unsigned document shown in
+<a href="#example-credential-without-proof-for-features"></a> and the same
+mandatory pointers shown in
+<a href="#example-mandatory-pointers-for-features"></a>. This results in the
+same canonical document as in
+<a href="#example-canonical-document-for-features"></a>, the same HMAC document
+as in
+<a href="#example-canonical-hmac-document-for-features"></a>, and the same "add
+base transformation" as in
+<a href="#example-add-base-transformation-for-features"></a>.
+          </p>
+          <p>
+This example makes use of the same proof configuration as
+<a href="#example-base-proof-configuration-for-features"></a>.
+This results in the same canonical base proof as in
+<a href="#example-canonical-base-proof-configuration-for-features"></a>.
+Combining this with the above assumptions leads to the same base hashes as in
+<a href="#example-add-base-hashes-for-features"></a>.
+          </p>
+          <p>
+Since |featureOption| is equal to `"pseudonym_issuer_pid"` the procedure of
+section <a href="#base-proof-serialization-bbs-2023">Base Proof
+Serialization</a> will produce the output shown below. This makes use of the
+signature generation algorithm of [[CFRG-Pseudonym-BBS-Signature]]. Note the
+inclusion of the
+|pid| value as this needs to be communicated to the holder.
```

Comment:
```suggestion
Since |featureOption| is equal to `"pseudonym_issuer_pid"`, the procedure of
section <a href="#base-proof-serialization-bbs-2023">Base Proof
Serialization</a> will produce the output shown below. This makes use of the
signature generation algorithm of [[CFRG-Pseudonym-BBS-Signature]]. Note the
inclusion of the
|pid| value, as this needs to be communicated to the holder.
```